resources/build/build-utils.sh

```diff
@@ -190,7 +167,7 @@ function findShouldSentryRelease() {
     esac
 }
 
-findRepositoryRoot
+findKeymanRoot
 _builder_setBuildScriptIdentifiers
```

Comment:
This line needs to go into builder.inc.sh.

Reply:
I think 

### build-utils.sh
We kept `findKeymanRoot`

### builder.inc.sh
` _builder_init()` calls `_builder_find_repoRoot`